Week 6 spencer

spencer-campbell opened the pull request:
Work I did during week 6. In the future I imagine gathering consent from the whole cooperative, for updates to our website. At the moment, I'm going forward with the expectation that consent from @arneultroneous and I is good enough, so I ask him to review my work.

As an apology: I don't feel finished with the work I started, here. It's messy, and parts are inaccurate. In the domain of documenting patterns, I haven't yet found a rhythm of opening and closing tasks that fits my weekly life.

Nevertheless, I'm sending it for approval, in the spirit of moving forward, accepting past mistakes, and improving.

arneultroneous commented on source/_data/patterns.yml:
```diff
@@ -1,127 +1,143 @@
 - name: "practising worldlove"
-  elements:
+  expectations:
     - "I leave my 'workspace'."
```
Okay I understand that it is documented here now. Good. :). I don't really understand why this is in "roles".
And yes I know you said you are not satisfied with your work. I know I write comments like you didn't say this in the beginning. The purpose is just clear communication.

arneultroneous replied:
I clicked on start review. what happens now?

spencer-campbell replied:
Start review is pretty neat, because it invites us to write replies.

spencer-campbell commented:
"Leading": part of an experiment. I just added all those there as kind of placeholder roles, so that I could change the structure, but then there wasn't any content to fill the new structure, so I changed it back.

"Arriving in celebration": I deleted this one because I thought it no longer accurately represented the way we've been doing things. Recently we arrive with a simple meditation, not telling stories, and the storytelling part is included in the celebrating itself -- which then also involves writing notes. Now I would probably write three patterns: "meeting on weekends", and, as subpatterns of it, "meditating to arrive", "celebrating the last week". Something like that.

"Managing knowledge": it's out. I think it's too big for us to write it, at this stage. Better to document smaller parts of it that are concretely part of what we do now. Like, for example, working with github.

One-minute praising to Arne for the clarifying questions. I feel like this recovers more value from my half-finished work, by giving me a space to say what my intentions were.

DioAlma commented:
I just found all of this now.
Will report ASAP.
